ci/slack_user_ids

```diff
@@ -6,3 +6,6 @@ shayne@shaynefletcher.org UBRBXAJHY
 stefano.baghino@digitalasset.com UD3MUK6TA
 nick.chapman@digitalasset.com UHHMLCNGM
 remy.haemmerle@daml.com UEGT3JT0Q
+sofia.faro@digitalasset.com U011H6KFYPN
+231829+associahedron@users.noreply.github.com U011H6KFYPN
```

Comment:
When do you commit with this?

Reply:
That's the email associated with this account, @associahedron

Reply:
Oh I see :+1: